resources/i18n/en.json

```diff
@@ -175,5 +175,7 @@
     "zim-name": "Zim Name",
     "zim-path": "Zim File Path",
     "export-reading-list": "Export reading list",
-    "export-reading-list-error": "An error has occured during export."
+    "export-reading-list-error": "An error has occured during export.",
+    "import-reading-list": "Import reading list",
+    "import-reading-list-error": "An error has occured during import."
```

Comment:
@kelson42 Are you fine with such an error message? I don't like that "reading list" is missing from the error text. Also no details about the error are provided, but addressing that one will require changes to `libkiwix`, and most likely that part won't be translated.

Reply:
Changed the message as requested and waiting for confirmation.